django-stubs/utils/termcolors.pyi

```diff
@@ -1,14 +1,14 @@
 from collections.abc import Callable, Mapping, Sequence
 from typing import Any
 
-color_names: Sequence
-foreground: Mapping[str, str]
-background: Mapping[str, str]
+color_names: tuple[str]
+foreground: dict[str, str]
+background: dict[str, str]
```

Comment:
The available colors seem to be fixed, should we switch to:

```suggestion
color_names: Final[tuple["black", "red", "green", "yellow", "blue", "magenta", "cyan", "white"]]
foreground: # Some kind of typed dict
background: # Same
```

Reply:
Why `Final`? I think it should be `tuple[Literal[...], ...]`

Reply:
So that users don't actually set these variables to something else. But it's fine without it as well, I feel `Final` conveys a stronger meaning (e.g. with `numpy` and `np.PI`).

Reply:
Ended up keeping `tuple[str]`, literals feel a bit overkill